client/jetpack-cloud/sections/golden-token/golden-token-dialog/index.tsx

```diff
@@ -0,0 +1,141 @@
+import classnames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import { useRef, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { getCurrentUserName } from 'calypso/state/current-user/selectors';
+
+import './style.scss';
+
+export const GoldenTokenDialog = () => {
+	const translate = useTranslate();
+	const userName = useSelector( getCurrentUserName );
+
+	const videoRef = useRef< HTMLVideoElement >( null );
+
+	const [ isAnimating, setIsAnimating ] = useState( false );
+
+	const redeemToken = () => {
+		if ( videoRef.current ) {
+			videoRef.current.play();
+		}
+		setIsAnimating( true );
+	};
+
+	return (
+		<div className={ classnames( 'golden-token-dialog', { animating: isAnimating } ) }>
+			<svg
```

Comment:
For cleanliness, would it make sense to make this SVG its own component?

Reply:
Yes, for this and all the others I plan to go back through and switch them to existing icon components.